File: config-server/src/main/java/com/greenlight/configserver/ConfigServerLauncher.java
package com.greenlight.configserver;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ConfigServerLauncher {
  public static void main(String[] args) {
    SpringApplication.run(ConfigServerLauncher.class, args);
  }

  public void s() {
    if (true) {
      System.out.println("test SQ analytics");
    }
  }
}

File: config-server/src/main/kotlin/com/greenlight/configserver/ConfigServerLauncher.kt
package com.greenlight.configserver

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.config.server.EnableConfigServer

@EnableConfigServer
@SpringBootApplication
class ConfigServerLauncher

fun main(args: Array<String>) {
    runApplication<ConfigServerLauncher>(*args)
}